### DISCUSSION TITLE
求助！对接光华平台

### DISCUSSION THREAD

**Discussion by dzhiqin** [NervJS/taro#13074] [官方公告]
光华平台是小程序的埋点监控平台，该平台提供的接入demo是用了支付宝小程序的原生写法，[参考链接](https://opendocs.alipay.com/pre-open/01rm95#4.2.2%20%E5%BC%95%E5%85%A5%E5%85%89%E5%8D%8E%E5%B9%B3%E5%8F%B0%E6%8F%92%E4%BB%B6js%E6%96%87%E4%BB%B6) 。
不知道用taro的react写法应该怎么写？
```
const monitor = require('/utils/alipayLogger.js');

App(monitor.hookApp({
  onLaunch(options) {
    // 第一次打开
    // options.query == {number:1}
    console.info('App onLaunch', options);

    ////////// 政务标准H5数据上报
    monitor.init({
      pid: "TODO",      // TODO,
      options: options,
      sample: 1,
      autoReportApi: true,
      // autoReportApi: false,
      autoReportPage: false,
      // autoReportPage: false,
      // Http请求返回数据中状态码字段名称
      code: ["code"],
      // Http返回数据中的error message字段名称
      msg: ["msg"]
    });
    //////////////////////
  },
  onShow(options) {
    // 从后台被 scheme 重新打开
    // options.query == {number:1}
  },
}));

```

我项目的taro版本是3.3.12

**Comment by dzhiqin** [comment]
附上alipayLogger和cloudMonitorHelper的代码
[alipayLogger](https://www.wolai.com/leshar/eqAHzmHSKmzVGuniST6Dbk)
[cloudMonitorHelper](https://www.wolai.com/leshar/ipk2tU1mGCre8vWKowgevX)

**Comment by tourze** [comment]
看逻辑，应该直接app.js import然后onLaunch照常执行就好了吧

**Reply by dzhiqin** [reply]
> 看逻辑，应该直接app.js import然后onLaunch照常执行就好了吧

试了下确实可行，谢谢
